tests/tests.js

```diff
@@ -589,10 +589,8 @@ exports.defineAutoTests = function() {
                 removeContactsByFields(["note"], "DeleteMe", function() {
                     contact.save(function(c_obj) {
                         var findWin = function(cs) {
-                            // update to have proper saved id
-                            contact = cs[0];
                             expect(cs.length).toBe(1);
-                            done();
+                            done(cs[0]);
```

Comment:
The docs don't say anything about it (but the Jasmine docs are also pretty lackluster). My guess is that the parameter would just get ignored.